index.html

```diff
@@ -29,19 +29,13 @@
                         "Please enter a name for the new document:",
                     )
                     var newDoc = {id: 42, name: name, content: ""} // FIXME
-                    setActive(newDoc.id)
                     setDocuments([...documents, newDoc])
-
-                    // Doesn't seem to work?
-                    setCurrentContent(
-                        documents.find((d) => d.id == newDoc.id).content,
-                    )
+                    setActive(newDoc.id)
                 }
 
-                // Seems to crash:
                 function deleteDocument(id) {
                     setActive(null)
```

Comment:
There is no document with `id: null`—which made `currentContent` crash (this is fixed now below).